roles/confluent.test/molecule/mtls-java8-rhel/molecule.yml

```diff
@@ -95,7 +95,7 @@ provisioner:
         ssl_enabled: true
         ssl_mutual_auth_enabled: true
 
-        redhat_java_package_name: java-11-openjdk
+        redhat_java_package_name: java-1.8.0-openjdk
```

Comment:
Another thing to consider is the dockerfiles we use for molecule are installing java 8, and then cp-ansible will install java 11 on top, which is a waste... I do like that these are keeping java 8 tests around. I'd say the simplest is just to update the dockerfiles to install java 11 and these scenarios can go without a dockerfile or something

Reply:
Good catch and nice suggestion! I will try to fix this.

Reply:
I have updated the dockerfiles to install Java11 and also the `certificates.yml` playbook - which seems to be used in the prepare phase of multiple scenarios. The non-default Java scenarios (now Java 8) does not seem to have custom dockerfiles. PTAL!